bin.src/comcamCloseLoop.py

```diff
@@ -326,6 +326,13 @@ def _useMinDofIdx(ofcCalc):
 
     ztaac.setZkAndDofInGroups(m1m3Bend=m1m3Bend, m2Bend=m2Bend)
 
+def eraseFolderContent(targetDir):
+    for the_file in os.listdir(targetDir):
```

Comment:
It might be better to use `theFile` instead of `the_file` to be consistent with other variable names. For example, you use `targetDir` already as the lower-case-first camel form.